src/Elastic.Documentation.Configuration/TableOfContents/ITocItem.cs

```diff
@@ -22,4 +22,7 @@ public record TocReference(Uri Source, ITableOfContentsScope TableOfContentsScop
 {
 	public IReadOnlyDictionary<Uri, TocReference> TocReferences { get; } =
 		Children.OfType<TocReference>().ToDictionary(kv => kv.Source, kv => kv);
+
+	public bool IsPhantom { get; init; }
```

Comment:
Can you please add a comment what `IsPhantom` means? 🙏

Reply:
Good callout! I did my best to explain their purpose in https://github.com/elastic/docs-builder/pull/1353/commits/05f89cce44699b4e5b30bd257d5b3a84c87b1234